src/main/main.ts

```diff
@@ -627,6 +626,20 @@ async function initializeHeadless(): Promise<void> {
   */
   console.log(`Initialize headless. (win: ${win?.getTitle})`);
 
+  if (initializeHeadlessCts !== null) {
+    console.error(
+      "Cannot initialize headless while initializing headless.",
+    );
+    return;
+  }
+
+  if (standalone.alive) {
```

Comment:
Can you change `standalone` to `headless`?

Reply:
It would be large diff including file name change, I guess it is better to be in another PR.